en/docs/develop/instagram-authenticator.md

```diff
@@ -1,9 +1,7 @@
 # Configuring Instagram Authenticator
 
 !!! note
-        This authenticator is now deprecated as the Instagram API 
-	Platform which supported authentication is no longer available. 
-	The [Facebook Authenticator](../../learn/configuring-facebook) can be used as an alternative to this.
+	This authenticator is now deprecated as the Instagram API Platform which supported authentication is no longer available. The [Facebook Authenticator](../../learn/configuring-facebook) can be used as an alternative to this.
 
```

Comment:
What are we changing here?

Reply:
We are aligning the sentence. 
Refer the note in https://is.docs.wso2.com/en/next/develop/instagram-authenticator/.